sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1136,6 +1140,20 @@ static int sof_card_dai_links_create(struct device *dev,
 		INC_ID(be_id, cpu_id, link_id);
 	}
 
+	if (sof_sdw_quirk & SOF_SSP_BT_PCM) {
+		ssp_components = devm_kzalloc(dev, sizeof(*ssp_components),
+						GFP_KERNEL);
+		if (!ssp_components)
+			return -ENOMEM;
+		/* BT to SSP is fixed on Port 2 for TGL/ADL platforms */
+		ssp_components->name = "snd-soc-dummy";
+		ssp_components->dai_name = "snd-soc-dummy-dai";
+		cpus[cpu_id].dai_name = "SSP2 Pin";
+
+		init_dai_link(dev, links + link_id, be_id, "SSP2-NoCodec", 1, 1,
+				cpus + cpu_id, 1, ssp_components, 1, NULL, NULL);
```

Comment:
@yongzhi1 do you mind updating the code to use the same convention as with tgl-max98373 (SSP2-BT) and quirks? Thanks.

Reply:
Sure, sorry about the delayed response.